ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml

```diff
@@ -147,6 +147,26 @@ keys:
                 - int
               no_autoconfig_flag:
                 type: bool
+        ipv6_dhcp_relay_destinations:
+          type: list
+          primary_key: address
+          items:
+            type: dict
+            keys:
+              address:
+                type: str
+                description: DHCP server's IPv6 address
+              vrf:
+                type: str
+              local_interface:
```

Comment:
Maybe add in the description that source_address and local_interface should not both be used (until we have a better mechanism to do it!)

in particular because in the template you are using if/elif and so we are giving better precedence to local_interface

```
[no|default] ipv6 dhcp relay destination IP6ADDR [ ( VRF_VRF_KW ( VRF_VRF_DYNAMIC ) ) ] [ ( local-interface INTF ) | ( source-address ADDR ) ] [ link-address LINKADDR ]
```

Reply:
Added in clarification that these are mutually exclusive. Thanks!